web/admin/src/pages/document/component/ImportDoc/Confluence.tsx

```diff
@@ -95,37 +103,67 @@ const ImportDocConfluence = ({
     setCurrentFileIndex(0);
     try {
       for (let i = 0; i < acceptedFiles.length; i++) {
-        const pages = await postApiV1CrawlerConfluenceAnalysisExportFile({
+        const { id = '', docs = [] } = await postApiV1CrawlerConfluenceParse({
           file: acceptedFiles[i],
           kb_id,
         });
-        for (const page of pages) {
+        setId(id);
+        for (const page of docs) {
           setItems(prev => [
             {
-              url: page.id! as unknown as string,
+              url: page.id!,
               title: page.title!,
-              content: page.content!,
+              content: '',
               success: -1,
               id: '',
             },
             ...prev,
           ]);
         }
       }
-      setStep('import');
+      setStep('pull-done');
       setLoading(false);
     } catch (error) {
       console.error(error);
     }
   };
 
+  const handleSelectedExportedData = async () => {
+    const newQueue: (() => Promise<any>)[] = [];
+    const confluenceData = items.filter(item => selectIds.includes(item.url));
+    for (const item of confluenceData) {
+      newQueue.push(async () => {
+        const res = await postApiV1CrawlerConfluenceScrape({
+          id,
+          doc_id: (item as any).url || '',
```

Comment:
在处理URL时，item.url被强制转换为any类型，这可能导致类型安全问题。建议添加类型检查或使用类型断言来确保类型安全。

```suggestion
const url = (item as ImportDocListItem).url || '';
// 或者
const url = item.url as string || '';